example/plugins/backends/saml2_backend.yaml.example

```diff
@@ -13,6 +13,8 @@ config:
       local: [idp.xml]
 
     entityid: <base_url>/<name>/proxy_saml2_backend.xml
+    idp_blacklist_enabled: true
```

Comment:
Do we really need this? An undefined or empty `idp_blacklist_file` entry would suffice, imo.